spec/acceptance/ntp_install_spec.rb

```diff
@@ -1,11 +1,11 @@
 require 'spec_helper_acceptance'
 
-case fact('osfamily')
-when 'FreeBSD'
+case os[:family]
+when 'freebsd'
```

Comment:
I would say you could remove both this and AIX? Since they aren't supported for ntp?

Reply:
Will remove FreeBSD occurrences in another PR.